data/categories.txt

```diff
@@ -899,16 +899,16 @@ sw:Baa|vinywaji
 fa:میکده|کاباره
 
 amenity-cafe|@eat
-en:3Cafe|3restaurant|U+2615|U+1F356|U+1F357|U+1F35A|U+1F35B|U+1F35C|U+1F35D|U+1F363|U+1F366|U+1F367|U+1F368|U+1F369|U+1F370|U+1F372|U+1F374|U+1F377|U+1F60B|U+1F375|bistro|lunch|coffee shop|coffee bar|coffee|canteen|cafeteria|dining room|place to eat|buffet
-ru:3Кафе|3ресторан|кофейня|столовая|кофе
-bg:3Кафе|3ресторант|кафене|столова|бюфет|заведение|ядене|храна
+en:3Cafe|3restaurant|U+2615|U+1F356|U+1F357|U+1F35A|U+1F35B|U+1F35C|U+1F35D|U+1F363|U+1F366|U+1F367|U+1F368|U+1F369|U+1F370|U+1F372|U+1F374|U+1F377|U+1F60B|U+1F375|coffee|cafeteria
```

Comment:
Let's remove restaurant from cafe, and cafe from restaurant. Initially, when I added them in 2011, I didn't know how important is to differentiate them.